Dockerfile-production

```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```

Comment:
@mleanos @codydaig that gem install isn't required anymore for gulp.
I deprecated it when I introduced the node-sass library which is purely C based and doesn't need the ruby gem dependency